src/main/webapp/app/core/auth/account.service.ts

```diff
@@ -26,6 +27,41 @@ export class AccountService {
   keycloakService = inject(KeycloakService);
   userResourceService = inject(UserResourceService);
 
+  /**
+   * Returns the full name of the signed-in user, or undefined if no user is loaded.
```

Comment:
Please update this comment to accurately reflect the purpose of this getter
```suggestion
   * Returns the id of the signed-in user, or undefined if no user is loaded.
```